### PR TITLE
Add checkbox questions to the browser tests

### LINKED ISSUE #707: Exporters need to handle scalar lists
**Describe the bug**
The [`CsvExporter`](https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/services/export/CsvExporter.java) currently uses `ApplicantData#readString` to read each value. However, this won't work for scalars that are actually lists, like how we store multiple selections for checkboxes (an error is thrown by JsonPath).

**Expected behavior**
We need a method that handles non-string json types and turns them into strings for export

**Additional context**
Found in `ProgramApplicationView` - lists appear `<unanswered>`

### PR DESCRIPTION
**Pull request by carolinedanzi**
### Description
1. Adds a function to create a checkbox question in the admin flow
2. Adds a function to fill out a checkbox question as an applicant
3. Adds a `readAsString` method to `ApplicantData` and calls it from `ProgramApplicationView`, `PdfExporter`, and `CsvExporter`

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issues
Fixes #707

### CODE REVIEW

File: universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java [hunk 1]
```diff
@@ -326,6 +326,32 @@ private void putParentArray(Path path) {
...
+   * Reads the value at the given path as a string. Returns {@link Optional#empty} if there is no
+   * value at the path. For JSON arrays of strings, this formats the array as a string according to
+   * {@link ImmutableList#toString}.
+   *
+   * @param path the {@link Path} to read
+   * @return optionally returns the value at the path as a string if it exists, or empty if not
+   */
+  public Optional<String> readAsString(Path path) {
+    if (isJsonArray(path)) {
+      return readList(path).map(ImmutableList::toString);
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java:339]: (nonblocking) could this not always map `toString`?

Comment (kendrickt) [reply universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java:339]: Ahh i see in the tests it packs numerical values into strings.

**Review by kendrickt — APPROVED** [review]
looks good to me

File: universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java (hunk 1 above)
Comment (carolinedanzi) [reply universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java:339]: Yeah every other type is fine - it just doesn't understand JSON arrays or objects (but  we don't have objects yet)

Thanks for the review!

**Pull request merged**